runtime/src/accounts_db.rs

```diff
@@ -175,6 +175,13 @@ pub enum AccountStorageStatus {
     Candidate = 2,
 }
 
+#[derive(Debug)]
+pub enum BankHashVerificatonError {
```

Comment:
In my opinion, the naming here is very clear after all the hustle of naming changes. :)